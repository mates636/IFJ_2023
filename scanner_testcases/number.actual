token type: 13 
token data: "2.22" 
token type: 36 
token type: 11 
