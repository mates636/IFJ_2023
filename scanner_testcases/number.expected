10 "2.22"
32
8
